.isort.cfg

```diff
@@ -2,7 +2,6 @@
 line_length=79
```

Comment:
According to the `black` profile, we should be able to get ride of the following options:
```
multi_line_output=3
include_trailing_comma=1
force_grid_wrap=0
use_parentheses=1
```
https://github.com/PyCQA/isort/blob/377d260ffa6f746693f97b46d95025afc4bd8275/isort/profiles.py#L4-L11